docs/current/cookbook/snippets/cache-invalidation/main.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import anyio
+import datetime
+import dagger
+
+async def main():
+
+    # create Dagger client
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr, workdir=".")) as client:
+
+        # invalidate cache to force execution
+        # of second with_exec() operation
+        output = await (
+            client.pipeline("test").
+            container().
+            from_("alpine").
+            with_exec(["apk", "add", "curl"]).
+            with_env_variable("CACHEBUSTER", str(datetime.datetime.utcnow().timestamp())).
```

Comment:
This can be simplified to:
```suggestion
            with_env_variable("CACHEBUSTER", str(datetime.now())).
```
Imported as
```python
from datetime import datetime
```

Reply:
I'll keep it to datetime, for consistency with the Node and Go versions...unless you have similar suggestions for those? Added in https://github.com/dagger/dagger/pull/5189

Reply:
I don't think we need consistency here because being idiomatic is more important. The timestamp factors into the uuid calculation btw. Since this is to bust the cache there's no advantage in getting the same value (and these implementation don't anyway), just the simplest randomly unique built-in you can use.